lib/state/ui/reducer.ts

```diff
@@ -1,11 +1,24 @@
 import { difference, union } from 'lodash';
 import { AnyAction, combineReducers } from 'redux';
+
 import * as A from '../action-types';
 import * as T from '../../types';
 
 const defaultVisiblePanes = ['editor', 'noteList'];
 const emptyList: unknown[] = [];
 
+const editMode: A.Reducer<boolean> = (state = true, action) => {
```

Comment:
changing from named states to boolean seems like a semantic regression.

does `true` or `false` mean "preview"?

I can follow the idea if we're saying "edit mode" _is_ the mode, though particularly since we talk about edit vs. preview when discussing this I personally find `'edit'` and `'preview'` clearer in the code

Reply:
I reasoned about this for a while.  I'd love to chat about it but I came to the opposite conclusion. We are in either editor more or preview.  If edit mode is true then we are in preview.  Just as is the tag panel open or closed, or are we in focus mode or is the note list open.